.github/workflows/ur-build-hw.yml

```
@@ -0,0 +1,132 @@
+name: UR - Build adapters on HW
```

maybe a nit, but what does it mean to build an adapter on hardware?

---

In this case it means run the tests on hardware, these jobs don't currently have a build/test separation. The name can be updated:

```suggestion
name: UR - Build adapters, test on HW
```